source/jormungandr/jormungandr/schedule.py

```diff
@@ -63,10 +63,11 @@ def get_realtime_system_code(route_point):
 
 
 class RealTimePassage(object):
-    def __init__(self, datetime, direction=None, is_real_time=True):
+    def __init__(self, datetime, direction=None, is_real_time=True, stop_id=None):
```

Comment:
Because **RealTimePassage** can be have different _stop_id_ now. Just for confort, and maybe for future feature. It does not seem very expensive in terms of memory.
Anyway, I use it in test : https://github.com/CanalTP/navitia/pull/2936/files#diff-c8ddb4b283f7492e8aba70cf9c7c4e71R474